src/com/yetanalytics/lrs_admin_ui/db.cljs

```diff
@@ -90,7 +141,9 @@
                           ::notifications
                           ::enable-statement-html
                           ::oidc-auth
-                          ::oidc-enable-local-admin]))
+                          ::oidc-enable-local-admin
+                          ::enable-admin-status
```

Comment:
Side note but it would be nice if all these "enable" booleans get put into a map that's nested under the top level.

Reply:
Just for organization's sake sure. Well-written subs should ensure this doesn't cause extra renders tho.